client/src/main/java/com/mytiki/publish/client/auth/AuthToken.kt

```diff
@@ -1,11 +1,10 @@
 package com.mytiki.publish.client.auth
 
-import android.util.Log
 import com.mytiki.publish.client.email.EmailProviderEnum
 import org.json.JSONObject
 import java.util.Date
 
-class AuthToken (val username: String, val auth: String, val refresh: String, val expiration: Date, val provider: EmailProviderEnum){
+class AuthToken (val email: String, val auth: String, val refresh: String, val expiration: Date, val provider: EmailProviderEnum){
```

Comment:
maybe we should keep it as username or user... we will use the authtoken for the TIKI tokens too.